Makefile

```diff
@@ -1,24 +1,49 @@
-REQUIREMENTS=requirements.txt
-REQUIREMENTS_CEXT=requirements/cext.txt
-REQUIREMENTS_CEXT_NOARCH=requirements/cext_noarch.txt
-
 .PHONY: help clean clean-pyc clean-build list test test-all coverage docs release sdist
 
 help:
-	@echo "clean-build - remove build artifacts"
-	@echo "clean-pyc - remove Python file artifacts"
-	@echo "lint - check style with flake8"
-	@echo "test - run tests quickly with the default Python"
-	@echo "testall - run tests on every Python version with tox"
-	@echo "coverage - check code coverage quickly with the default Python"
-	@echo "docs - generate Sphinx HTML documentation, including API docs"
-	@echo "release - package and upload a release"
-	@echo "dist - package"
-	@echo "writeversion - updates the kolibri/VERSION file"
-
-clean: clean-build clean-pyc clean-docs clean-static
-
-clean-static:
+	@echo "Usage:"
+	@echo ""
+	@echo "make <command>"
+	@echo ""
+	@echo "Building"
+	@echo "--------"
+	@echo ""
+	@echo "dist: create distributed source packages in dist/"
+	@echo "pex: builds a portable .pex file for each .whl in dist/"
+	@echo "assets: builds javascript assets"
+	@echo "staticdeps: downloads/updates all static deps bundled into the dist"
```

Comment:
can we clarify: "all static deps" -> "all static Python dependencies" or similar?